.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Closes # (issue)
+<!-- REMOVE ALL THE TEMPLATE BELOW IF THE PR IS A RELEASE -->
+# Checklist:
+<!-- CHECKLIST_TYPE: ALL -->
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+<!-- /CHECKLIST_TYPE -->
+
+# Type of change. This type of change is the main reason for the PR.
+<!-- CHECKLIST_TYPE: ONE -->
+- [ ] FIX - a PR of this type patches a bug.
+- [ ] FEATURE - a PR of this type introduces a new feature.
+- [ ] BUILD - a PR of this type introduces build changes.
+- [ ] CI - a PR of this type introduces CI changes.
+- [ ] DOCS - a PR of this type introduces DOCS improvement.
+- [ ] STYLE - a PR of this type introduces style changes.
+- [ ] REFACTOR - a PR of this type introduces refactoring.
+- [ ] PERFORMANCE - a PR of this type introduces performance changes.
+- [ ] TEST - a PR of this type adds more tests.
+- [ ] CHORE - a PR introduces other changes than the specified above.
```

Comment:
There's an overlap in changes here. For example #166 introduces a new **feature** (the Vue example itself) while also updating **docs** and **refactoring** the existing React example.

I appreciate we want to keep PRs concise but maybe we should make this list smaller or allow multiple choices?

Reply:
I'm guessing it has to be one item for Conventional Commits? The description does say "main reason" so maybe in the case of #166, I'd just tick FEATURE and not overthink it.